plugins/outputs/redis/redis.go

```diff
@@ -0,0 +1,126 @@
+package redis
+
+import (
+	"fmt"
+	"time"
+
+	redigo "github.com/garyburd/redigo/redis"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/plugins/serializers"
+)
+
+//use the redis service LIST struct as telegraf output
+type RedisOutput struct {
+	Addr     string `toml:"server_addr"`
+	Password string `toml:"server_passwd"`
+	Queue    string `toml:"queue_name"`
+
+	server     *redigo.Pool
+	serializer serializers.Serializer
+}
+
+var sampleConfig = `
+  ## redis service listen addr:port, default 127.0.0.1
+  # server_addr = "127.0.0.1:6379"
+  ## redis service login password
+  # server_passwd = ""
```

Comment:
Change option name to just `password`, this matches with other existing plugins.